README.md

```diff
@@ -23,20 +23,20 @@ Documentation
 
 
 * [partitioninfo](#module_partitioninfo)
-    * [.get(image, number)](#module_partitioninfo.get) ⇒ <code>Promise.&lt;Object&gt;</code>
-    * [.getPartitions(image, options)](#module_partitioninfo.getPartitions) ⇒ <code>Promise.&lt;Object&gt;</code>
+    * [~get(image, number)](#module_partitioninfo..get) ⇒ <code>Promise.&lt;Object&gt;</code>
+    * [~getPartitions(image, options)](#module_partitioninfo..getPartitions) ⇒ <code>Promise.&lt;Object&gt;</code>
```

Comment:
`.` → `..` looks intentional. If so, what does it mean?

Reply:
This is due to the update of `jsdoc-to-markdown`